x-pack/plugins/observability/public/pages/cases/all_cases.tsx

```diff
@@ -10,22 +10,18 @@ import React from 'react';
 import { AllCases } from '../../components/app/cases/all_cases';
 import * as i18n from '../../components/app/cases/translations';
 
-import { permissionsReadOnlyErrorMessage, CaseCallOut } from '../../components/app/cases/callout';
 import { CaseFeatureNoPermissions } from './feature_no_permissions';
+import { HeaderBadge } from './header_badge';
 import { useGetUserCasesPermissions } from '../../hooks/use_get_user_cases_permissions';
 import { usePluginContext } from '../../hooks/use_plugin_context';
 
 export const AllCasesPage = React.memo(() => {
   const userPermissions = useGetUserCasesPermissions();
   const { ObservabilityPageTemplate } = usePluginContext();
+
   return userPermissions == null || userPermissions?.read ? (
     <>
-      {userPermissions != null && !userPermissions?.crud && userPermissions?.read && (
```

Comment:
The design team has requested that we no longer show callouts for the readonly user. Instead we're going to show the readonly badge.